.github/workflows/cronet.yml

```diff
@@ -19,48 +21,45 @@ env:
   PUB_ENVIRONMENT: bot.github
 
 jobs:
-  analyze:
-    name: Lint and static analysis
-    runs-on: ubuntu-latest
```

Comment:
Why did you change this to macos? I think that there is more linux machine availability.

Reply:
There are unknown timeouts when running on the Ubuntu runner. https://github.com/dart-lang/http/actions/runs/7270023551/job/19808487300?pr=1070